Lesson9/merve/index.html

```diff
@@ -0,0 +1,141 @@
+              <a class="btn" rel="noopener follow" href="https://medium.com/m/signin">Follow</a>
+        </div>
+        <div class="next-info">
+           <span class="light">Published in</span>
+              <a class="link" href="https://medium.com/free-code-camp"
+                >We’ve moved to freeCodeCamp.org/news</a>
+              <span class="light"> · 3 min read · Aug 12, 2018</span>
+        </div>
+        <div class="divider"></div>
+        <div class="social-media-box">
+          <div class="left"></div>
+          <div class="right"></div>
+        </div>
+        <div class="divider"></div>
+        </div>
+        <img class="clock" loading="eager" role="presentation" src="https://miro.medium.com/v2/resize:fit:1250/1*aVzJTznRRfP1lM7AXe9yLw.jpeg" alt=""/>
+      <figcaption>Photo by <a class="link" href="">  &nbspKevin Ku</a>&nbspon<a class="link" href=""> &nbspUnsplash</a></figcaption>
+      <main>
+        <div class="article">
+          <p>As a web developer, there is a high probability that you have encountered the two enemies of this article: <b>images</b> and <b>deadlines.</b> Sometimes, for some reasons, your images won’t fit the layout and you don’t want to wrap your head around this for hours.</p>
+    <p>This situation has happened to me many times and I have learned from my mistakes. No more black magic hacks — here are my five favorites techniques to handle image resizing.</p>
+          <h2>the “OMG I NEED THIS ASAP” way</h2>
+    <p>It’s 5:00 pm on Friday, you have to finish this page, but the images won’t fit the layout. It’s time to use your magic trick!</p>
+        <pre>
+    .myImg{
+    background-image: url("my-image.png");
+    background-size: cover;
+    }</pre>
+          <p>Sounds familiar? We’ve all done it once, doesn’t it feel like cheating to you?</p>
+          <p>Using<cite> background</cite> properties is very useful, they just work. Yet, remember that you should use them only for non-content images or as a replacement of text and in <a href="">some particular cases</a>.</p>
+          <h3>The way from the future</h3>
+          <p>What if I told you this kind of magic exists also for<cite>&lt;img&gt;</cite>elements? Say “hi” to the <cite>object-fit</cite> property — which also works for videos, by the way!</p>
+          <pre>
+    .myImg{
+    object-fit: cover;
+    width: 320px;
+    height: 180px;
+    }</pre>
+           <p>
+            That’s all folks! See how when we retrieve the friendly value <cite>cover</cite>, we can also use <cite>contain</cite>.
+          </p>
+          <h4>Okay what’s the trap?</h4>
+          <p>Unfortunately <cite>object-fit</cite> will not work on IE and older versions of Safari, but there is a<a href=""> polyfill.</a></p>
+        </div>
+        <div class="images"><img alt="" class="chart" width="700" height="129" loading="lazy" role="presentation" src="https://miro.medium.com/v2/resize:fit:875/1*d0wZwFpXGiAYH9_NrJCroA.png"/></div> 
```

Comment:
Never add height and width to images directly in HTML, use CSS to define them. Same for other images in this file.